test/source/mock/google/google-endpoints.ts

```diff
@@ -138,76 +150,52 @@ export const getMockGoogleEndpoints = (oauth: OauthMock): HandlersDefinition =>
       }
       throw new HttpClientErr(`Method not implemented for ${req.url}: ${req.method}`);
     },
-    '/gmail/v1/users/me/settings/sendAs': async (parsedReq, req) => {
+    '/gmail/v1/users/me/settings/sendAs': async (_parsedReq, req) => {
       const acct = oauth.checkAuthorizationHeaderWithAccessToken(req.headers.authorization);
-      if (isGet(req)) {
-        const sendAs = [
-          {
-            sendAsEmail: acct,
-            displayName: 'First Last',
-            replyToAddress: acct,
-            signature: '',
-            isDefault: true,
-            isPrimary: true,
-            treatAsAlias: false,
-            verificationStatus: 'accepted',
-          },
-        ];
-        if (acct === 'flowcrypt.compatibility@gmail.com') {
-          sendAs[0].signature =
-            '<div dir="ltr">flowcrypt.compatibility test footer with an img<br><img src="https://flowcrypt.com/assets/imgs/svgs/flowcrypt-logo.svg" alt="Image result for small image"><br></div>';
-          const alias = 'flowcryptcompatibility@gmail.com';
-          sendAs.push({
-            sendAsEmail: alias,
-            displayName: 'An Alias',
-            replyToAddress: alias,
-            signature: '',
-            isDefault: false,
-            isPrimary: false,
-            treatAsAlias: false,
-            verificationStatus: 'accepted',
-          });
-        } else if (acct === 'multi.aliased.user@example.com') {
-          const alias1 = 'alias1@example.com';
-          const alias2 = 'alias2@example.com';
-          sendAs.push(
+      if (!isGet(req)) {
+        throw new HttpClientErr(`Method not implemented for ${req.url}: ${req.method}`);
+      }
+      const primarySendAs = {
+        sendAsEmail: acct,
+        displayName: 'First Last',
+        replyToAddress: acct,
+        signature: '',
+        isDefault: true,
+        isPrimary: true,
+        treatAsAlias: false,
+        verificationStatus: 'accepted',
+      };
+      // If the account is a compatibility account, return specific aliases.
+      // This account is used in hundreds of tests, so we handle it separately to avoid duplicating the code.
+      if (acct === 'flowcrypt.compatibility@gmail.com') {
+        const alias = 'flowcryptcompatibility@gmail.com';
+        return {
```

Comment:
@sosnovsky As you can see, when acct is `flowcrypt.compatibility@gmail.com`, I returned specific HARD_CODED aliases.
Do you agree with this or want me to duplicate below same configuration to every compatibility tests?

Reply:
It's ok to have it hardcoded here, no need for duplicated code